XCTWebDriverAgentLib/Commands/FBFindElementCommands.m

```diff
@@ -193,6 +193,7 @@ + (NSArray *)descendantsOfElement:(XCUIElement *)element withXPathQuery:(NSStrin
   NSMutableDictionary *elementStore = [NSMutableDictionary dictionary];
   DDXMLElement *xmlElement = [self XMLElementFromElement:element.lastSnapshot indexPath:@"top" elementStore:elementStore];
   NSError *error;
+  xpathQuery = [XCUIElement patchXPathQueryUIAClassNames:xpathQuery];
```

Comment:
Unfortunately yes. All UICollectionViewCells and UITableViewCells are represented as UIACellView, however xpath will still use UIACollectionViewCell etc